language/move-lang/tests/move_check/expansion/restricted_constant_names.move

```diff
@@ -18,3 +18,21 @@ module M {
 }
 
 }
+
+script {
+    const address: u64 = 0;
```

Comment:
It might be worth a comment in the test to explain that all of these except for "Self" are expected to fail because of starting with a lowercase letter, not because they are restricted names. The name of the test file suggests otherwise.